### PR TITLE
Update social fabric jan2022

### PR DESCRIPTION
**Pull request by wkhchow**
PR in sync with model factory update_social_fabric_jan2022 branch
https://github.com/OpenDRR/model-factory/pull/100

- update social fabric
- add new db_check()
- add MMI patch fix (I copied the 1 line of code from master to fix in this branch before the fix was applied to master so hopefully no conflicts here)
- model factory path branch will need to be updated after merge to master or assets

### CODE REVIEW

File: python/add_data.sh [hunk 1]
```diff
@@ -435,7 +435,7 @@ read_github_token() {
 # from the OpenDRR/model-factory repository
 get_model_factory_scripts() {
   # TODO: Make this more robust
-RUN git clone https://github.com/OpenDRR/model-factory.git --branch update_dsra_psra_oct2021 --depth 1 || (cd model-factory ; RUN git pull)
+RUN git clone https://github.com/OpenDRR/model-factory.git --branch update_social_fabric_jan2022 --depth 1 || (cd model-factory ; RUN git pull)
```
Comment (drotheram) [inline python/add_data.sh:438]: Change this back to pull a model-factory release

**Review by drotheram — CHANGES_REQUESTED** [review]
Looks good but before merging make sure a corresponding model-factory release is created and pull from that release rather than cloning from a branch or from main

**Review by DamonU2 — APPROVED** [review]

File: python/add_data.sh (hunk 1 above)
Comment (anthonyfok) [reply python/add_data.sh:438]: And this happen to be the very line that conflicts with the one in the `master` branch, so we'll have to resolve this conflict anyway.  😁  And looks like we'll need to do the same for future dual `model-factory`+`opendrr-api` PRs.

**Review by anthonyfok — APPROVED** [review]
Looks good to me!

Where possible, use "Amend commit..." or "Undo commit..." instead of "Revert changes in commit"... though apparently GitHub Desktop won't let you amend or undo when your local branch has been pushed to origin (or upstream)...

**Review by jvanulde — APPROVED** [review]

File: python/add_data.sh (hunk 1 above)
Comment (wkhchow) [reply python/add_data.sh:438]: Yup, let me try to adjust this section once the new model-factory is merge/released

Comment (wkhchow) [reply python/add_data.sh]: fixed

**Pull request merged**